src/search/serializers/paper.py

```diff
@@ -56,6 +60,23 @@ def get_unified_doc_id(self, paper):
         except Exception as e:
             log_error(e, 'A Paper must have unified document')
 
+    def get_uploaded_by(self, hit):
+        try:
+            paper = Paper.objects.get(id=hit['id'])
+            user = User.objects.get(
+                id=paper.uploaded_by.id
+            )
+            return UserSerializer(user, read_only=True).data
+        except Exception as e:
+            print(e)
+
+    def get_uploaded_date(self, hit):
+        try:
+            paper = Paper.objects.get(id=hit['id'])
+            return paper.uploaded_date
+        except Exception as e:
+            print(e)
```

Comment:
instead of printing these exceptions, can we log them? otherwise it'll be hard to find if something errors